### PR TITLE
Add a draft PR template w/ integration tests from Tom.

### PR DESCRIPTION
**Pull request by optimove-andy**
Adding a PR template so we have a standardised template, it also contains quite a lot of manual integration tests until we have more units wrapped around the new features.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+## T&T Only
+
+- [ ] Init SDK with only optimove tenant / config
+- [ ] Track events
+
+## Push Only
+
+- [ ] Init SDK with all credentials
+- [ ] Track events
+- [ ] Associate customer (verify both backends)
```
Comment (cgwyllie) [inline .github/pull_request_template.md:31]: Should this test be in T&T only too?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+- [ ] With app installed, trigger deep link handler
+- [ ] With app uninstalled, follow deep link, install test bundle, verify deep link read from Clipboard, trigger deep link handler
+
+## Combined
+
+- [ ] Track event for T&T, verify push received
+- [ ] Trigger scheduled campaign, verify push received
+- [ ] Trigger scheduled campaign, verify In-App received
+
+Release:
```
Comment (cgwyllie) [inline .github/pull_request_template.md:51]: Worth looking at heading levels on tests vs the steps? Perhaps we could just use bold for cases or something?

Are these steps complete -- not sure about how this SDK is published yet. (any tags to make etc.)

File: .github/pull_request_template.md (hunk 1 above)
Comment (cgwyllie) [reply .github/pull_request_template.md:31]: (suppose in other variant it's just verify other backend? or do we need to verify both?)

File: .github/pull_request_template.md (hunk 2 above)
Comment (optimove-andy) [reply .github/pull_request_template.md]: I assume they are not, @KonstantinOptimove can probably document / complete this step.

Comment (optimove-andy) [reply .github/pull_request_template.md]: For the headings i've done a pass to smooth out the heading levels.

**Pull request merged**